tests/acceptance/bootstrap/OcisConfigContext.php

```diff
@@ -75,6 +76,31 @@ public function theConfigHasBeenSetTo(string $configVariable, string $configValu
 		);
 	}
 
+	/**
+	 * @Given the administrator has enabled the permissions role :role
+	 *
+	 * @param string $role
+	 *
+	 * @return void
+	 */
+	public function theRoleHasBeenEnabled(string $role): void {
```

Comment:
method name should align with the step name
```suggestion
	public function theRoleHasBeenEnabled(string $role): void {
```

Reply:
Thanks for the suggestion.